### DISCUSSION TITLE
regex expressions in field validation

### DISCUSSION THREAD

**Discussion by thomaskamalakis** [Budibase/budibase#6549] [Help] [answered]
Dear all

  I am having trouble applying regex field validations containing OR conditions. 

  We want to write an expression to check a text field whether it is empty or it contains our internal domain @hua.gr

  So it is something like ^$|@hua.gr$

![image](https://user-images.githubusercontent.com/25413968/177181297-1feface8-d62c-4b39-8093-fb30f8fa8be1.png)

  However this does not seem to work:

![image](https://user-images.githubusercontent.com/25413968/177182524-388953a0-d0f3-487f-b4a0-75564d7721af.png)

  Here is the exported app
  
[My app-export-1656947904579.txt](https://github.com/Budibase/budibase/files/9040631/My.app-export-1656947904579.txt)

  Thanks!

**Comment by melohagan** [comment, marked as answer]
Hi @thomaskamalakis 

Thank you for bringing this to our attention, this is a bug which I've raised here: https://github.com/Budibase/budibase/issues/6553

It was a simple fix and I have a PR out for it, so that should hopefully be available in the next release in about a week!

**Comment by thomaskamalakis** [comment]
@melohagan thank you!
